core/modules/user/user.module

```diff
@@ -1772,6 +1790,19 @@ function user_page_title($account) {
   return is_object($account) ? user_format_name($account) : '';
 }
 
+/**
+ * Implements hook_init().
+ */
+function user_init() {
+  $router = menu_get_item();
```

Comment:
Any specific reason why we are not using `current_path()` here? If so, then lets please document why.